caps/templates/caps/base.html

```diff
@@ -83,7 +83,7 @@
     </div>
 
     {% include 'caps/includes/site-footer.html' %}
-
+    
```

Comment:
can we remove this purely whitespace change from the commit?

Reply:
Yep!